core/services/ocr2/plugins/ocr2keeper/config/config.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+// PluginConfig contains custom arguments for the OCR2Keeper plugin.
+type PluginConfig struct {
+	// MinIncomingConfirmations is the minimum number of confirmations required ty sync updates from keeper registry
+	MinIncomingConfirmations uint32 `json:"minIncomingConfirmations"`
```

Comment:
if we are not using this for now then I would vote to keep it empty.

Also we have a global env var https://docs.chain.link/docs/configuration-variables/#min_incoming_confirmations so might just be able to use that in future

Reply:
Done